packages/messenger/src/Messenger.ts

```diff
@@ -181,9 +181,24 @@ export class Messenger<
   Namespace extends string,
   Action extends ActionConstraint,
```

Comment:
Side note, but should `Action` and `Event` default to `never` so that if you don't supply any type parameters to `Messenger`, allowing TypeScript to infer them, you don't end up with a messenger that recognizes all actions and events?